modules/app.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-plusplus */
 /* eslint-disable radix */
+import navigators from '../nav.js'
 
 let myLibrary = [];
```

Comment:
- [ ] Good job so far 👍 However, there is still some work to do 🏃. I can see that you have put your root or the main file in the module folder (I mean the `app.js`) and I think it is not appropriate to place to place it. Normally, the main file is supposed to be in the root level of the project folder or if you have an src folder then you can place it there because it acts as the entry point for your modules. That means it's not supposed to be in the modules folder. It is just a convention. I would advise you to place it either in the root level of your application or in the `js folder` maybe. That way, it will make sense 😃 👍